modules/300-prometheus/templates/prometheus/longterm/prometheus.yaml

```diff
@@ -133,8 +133,18 @@ spec:
       checksum/kube-rbac-proxy: {{ include "helm_lib_kube_rbac_proxy_ca_certificate" (list . "") | sha256sum }}
   secrets:
   - prometheus-api-client-tls
-  {{- include "helm_lib_node_selector" (tuple . "monitoring") | nindent 2 }}
-  {{- include "helm_lib_tolerations" (tuple . "monitoring") | nindent 2 }}
+  {{- if .Values.prometheus.longtermNodeSelector }}
+  nodeSelector: {{ .Values.prometheus.longtermNodeSelector | toJson }}
```

Comment:
```suggestion
  nodeSelector: {{ .Values.prometheus.longtermNodeSelector | toYaml }}
```
Why json?

Reply:
Where have you fixed ?

Reply:
Oh, still have confusion with personal and project repository.
Fixed right now.